android-project/app/src/main/java/org/libsdl/app/SDLAudioManager.java

```diff
@@ -71,12 +71,12 @@ protected static String getAudioFormatString(int audioFormat) {
         }
     }
 
-    protected static int[] open(boolean isCapture, int sampleRate, int audioFormat, int desiredChannels, int desiredFrames, int deviceId) {
+    protected static int[] open(boolean records, int sampleRate, int audioFormat, int desiredChannels, int desiredFrames, int deviceId) {
```

Comment:
```suggestion
    protected static int[] open(boolean recording, int sampleRate, int audioFormat, int desiredChannels, int desiredFrames, int deviceId) {
```

Reply:
Ah, see, I replaced all the `iscapture` vars with `records` because `recording` might be read as "this is pulling in data _right now_ (it's _recording_!)."

Reply:
Well, that is the intent, to be recording when opened.. :)